kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableFeaturesSuite.scala

```diff
@@ -195,9 +198,94 @@ class DeltaTableFeaturesSuite extends DeltaTableWriteSuiteBase {
     }
   }
 
+  test("feature can be enabled via delta.feature prefix") {
```

Comment:
can we maybe test this in the same txn with a different feature enabled by the metadata? I think this is the case that the original code wouldn't have covered -- just to be thorough

Reply:
done (I added "delta.feature override and TableConfig populate necessary features") let me know if you were thinking of something else.